src/bun.js/api/bun/subprocess.zig

```diff
@@ -502,14 +511,18 @@ const Readable = union(enum) {
                 this.* = .{ .closed = {} };
                 return pipe.toJS(globalThis);
             },
-            .buffer => |buffer| {
+            .buffer => |*buffer| {
                 defer this.* = .{ .closed = {} };
 
-                if (buffer.len == 0) {
+                if (buffer.length() == 0) {
                     return JSC.WebCore.ReadableStream.empty(globalThis);
                 }
 
-                const blob = JSC.WebCore.Blob.init(buffer, bun.default_allocator, globalThis);
+                const own = buffer.takeSlice(bun.default_allocator) catch {
+                    globalThis.throwOutOfMemory() catch return .zero;
+                    unreachable;
```

Comment:
this line can be removed; `globalThis.throwOutOfMemory()` always returns an error and the catch expression is `noreturn`, so this unreachable is not analyzed.

please avoid using unreachable anywhere unless it is actually semantically analyzed but not possible to reach.

Reply:
I can't ignore the return, the Zig compiler won't let me.
```zig
_ = globalThis.throwOutOfMemory();
```

Reply:
it's the 'unreachable' line that can be removed (or changed to 'comptime unreachable')